extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -343,6 +343,27 @@ export class AmpA4A extends AMP.BaseElement {
 
     /** @protected {boolean} */
     this.isRelayoutNeededFlag = false;
+
+    /** @private {!Object<string, ?JsonObject>} */
+    this.jsonConfig_ = {};
+    const jsonAttribute = element.getAttribute('json');
+    if (jsonAttribute) {
+      let jsonConfig;
+      try {
+        jsonConfig = JSON.parse(jsonAttribute);
```

Comment:
Considering use tryJsonParse.   What was the decision on whether we should send the request if json is invalid?  I thought we did not.  Is that still enforced?